features/shopify-sections.md

```diff
@@ -0,0 +1,47 @@
+# Shopify Sections
+
+* [Shopify sections](shopify-sections.md#shopify-sections)
+    * [Introduction](shopify-sections.md#introduction)
+    * [How it works](shopify-sections.md#how-it-works)
+    * [Add new placement](shopify-sections.md#add-new-placements)
+
+
+
+## Introduction
+Nosto supports Shopify sections. You can add new placements through sections
+or drag-and-drop existing ones.
+
+
+```
```

Comment:
You can use an info box like the one on this page https://docs.nosto.com/techdocs/implementing-nosto/checking-your-setup#order
![image](https://user-images.githubusercontent.com/22770093/169786463-e7fa4e66-29fe-4235-be6a-da973c5b69d0.png)